docs/running_locally.md

```diff
@@ -51,21 +11,31 @@ cp .env.sample .env
 Simply, edit your `.env` and set `RPC_NODE_URL` to the chain you want the services running against.
 **Important Note:** Only L2 safes are supported in the setup provided. Change this at your own risk.
 
+Now you should be able to run
 
-## Step 2: Setup djando superusers
+```bash
+docker compose up
+```
+
+## Step 2: Setup Django superusers
 
 You will need to identify the ID or name of the containers using `docker ps`. To create the default super user for the Safe Config Service, we run the following command:
 
 ```bash
-docker exec safe-infrastructure_cfg-web_1 python src/manage.py createsuperuser --noinput
+docker exec safe-infrastructure-cfg-web-1 python src/manage.py createsuperuser --noinput
```

Comment:
I wouldn't specify the step to identify the id or name of the container since any action to any container needs it and there might be other ways to do it (up to the user). Suggestion:

```
docker exec <container> python src/manage.py createsuperuser --noinput
```

We can also link it to https://docs.docker.com/engine/reference/commandline/exec/

Additional question: why `--noinput`?

Reply:
Even though the container names are susceptible to be changed by docker, I think it is more clear to refer specifically in which container we are doing this. If I were to make this instruction generic, I would be moving away from the "step-by-step" nature that I wanted to imprint to this guide.

`--noinput` enforces that no further user interaction is required and errors if the configuration is not correct: https://docs.djangoproject.com/en/3.0/ref/django-admin/#django-admin-createsuperuser